### DISCUSSION TITLE
Conflict server x pool of ips

### DISCUSSION THREAD

**Discussion by hugoperes** [postalserver/postal#2600] [Help with using Postal]
Hello dear,

In Postal version 2.14 I am having problems with IP pool updates.

After changing some IPS in Netplan's configuration file (Ubuntu 20.04), certain Postal servers stopped sending emails anymore (they have Status PENDING). Then if I exclude the IP from the Pool and associate the server to another pool (with other IPs) the emails start to be sent.

Does anyone know how I can resolve this conflict between server x ips poll without having to delete and create a new server?

**Comment by willpower232** [comment]
Its possible that changing the configuration messes with the networking on the server.

If you change the IPs in netplan and then restart Postal, does that avoid the issue?

**Reply by hugoperes** [reply]
I decided! I had accidentally duplicated an IP4/ip6 pair in netplan and it was giving errors on subsequent IP pairs.
